doc/level1.rst

```diff
@@ -294,11 +295,11 @@ state specified here, regardless of what additional state they track.
     a :rfc:`User ID <4880#section-5.11>` that matches the message's
     ``From`` address.
 
-``peer_state[A].state`` semantics
+``peers[A].state`` semantics
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-The ``state`` variable of a particular peer's ``peer_state`` data is
-selected from a set range of values:
+The ``state`` value of a particular ``peers`` entry is selected from a
```

Comment:
The wording is awkward.  I propose:

  The value of the ``state`` attribute can be either: